### ISSUE TITLE
`import_failed` false positive with relative imports

### ISSUE THREAD

**Issue by mflova** [quora/pyanalyze#700]
Given the following mock project:

```
root/
└── lib/
    ├── a.py
    └── b.py
```

```python
# a.py
# Empty
```

```python
# b.py
import a
```
For this example I can perfectly run `b.py` from both `root/` or from `lib/`. However, when I launch `pyanalyze` from `root/`, I get `import_failed` code. Apparently, because it cannot import a from `b.py`

**Comment by JelleZijlstra** [comment]
This is probably fixable by setting the PYTHONPATH environment variable appropriately. Possibly pyanalyze itself could set up its import path better too.
